File: API/Cinema.DB/Models/Auditorium.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Cinema.DB.Models
{
    public class Auditorium
    {
        public Guid Id { get; set; }
        public int Number { get; set; }
        public int Rows { get; set; }
        public int SeatsInRow { get; set; }
        public int Capacity { get; set; }
    }
}

File: Angular/cinema/src/models/Auditorium.ts
import { Seat } from "./Seat";

export class Auditorium {
    id: string;
    number: number;
    rows: number;
    seatsInRow: number; 
    capacity: number;
    seats: Seat[];

    constructor() {
        this.id = '',
        this.number = 0,
        this.rows = 0,
        this.seatsInRow = 0,
        this.capacity = 0,
        this.seats = []
    }
}